spring-cloud-netflix-core/src/main/java/org/springframework/cloud/netflix/zuul/filters/AntPathZuulRouteMatcher.java

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright 2013-2014 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.springframework.cloud.netflix.zuul.filters;
+
+import lombok.extern.apachecommons.CommonsLog;
+import org.springframework.cloud.netflix.zuul.filters.ZuulProperties.ZuulRoute;
+import org.springframework.util.AntPathMatcher;
+import org.springframework.util.PathMatcher;
+
+import java.util.Map;
+import java.util.concurrent.atomic.AtomicReference;
+
+/**
+ * A base implementation of {@link ZuulRouteMatcher} that uses {@link AntPathMatcher} for matching the request URI
+ * with configured routes.
+ *
+ * @author Jakub Narloch
+ */
+@CommonsLog
+public class AntPathZuulRouteMatcher implements ZuulRouteMatcher {
```

Comment:
Is a whole new abstraction needed? Could the functionality move to a protected method in `ProxyRouteLocator`?